### PR TITLE
Reworking hero

### PR DESCRIPTION
**Pull request by diemol**
I redid the hero part so it looks "ok" when the screen width changes, I think there is still room for improvement on how the elements are located after each resizing (also if their size should be bigger/smaller). I took the JS changes from #32 since they fix completely the issue where elements just left the viewport. 

The animation is more subtle now, but still open for improvement. Also, all elements are moving based on Petre's suggestion (either move everything or nothing).

hero-reworked

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -70,38 +70,73 @@ $(document).ready(function () {
...
             const x = event.clientX - parallaxBox.offsetLeft
             const y = event.clientY - parallaxBox.offsetTop
+            mouseParallax(l1, c1left, c1top, x, y, 5)
+            mouseParallax(l2, c2left, c2top, x, y, 10)
+            mouseParallax(l3, c3left, c3top, x, y, 10)
+            mouseParallax(l4, c4left, c4top, x, y, 10)
+            mouseParallax(l5, c5left, c5top, x, y, 10)
+            mouseParallax(l6, c6left, c6top, x, y, 10)
+            mouseParallax(l7, c7left, c7top, x, y, 10)
+            mouseParallax(l8, c8left, c8top, x, y, 10)
```
Comment (christian-bromann) [inline assets/js/script.js:125]: from the original the last value always differs from the other, not sure what it exactly made it different (I assume the floating area),

**Review by christian-bromann — DISMISSED** [review]
As per @billmcgee request we don't want to keep the robots moveable which is why c6 and c8 where commented out.

**Comment by diemol** [comment]
> As per @billmcgee request we don't want to keep the robots moveable which is why c6 and c8 where commented out.

Yes, that was the suggestion. But as I wrote in the PR description, Petre's suggestion is to either move all elements or move none of them.

File: assets/js/script.js (hunk 1 above)
Comment (diemol) [reply assets/js/script.js:125]: The last number is the input to some formula where the element oscilates, the smaller the number, the more subtle the movement. I described above that this is the intention, more subtle movements.

File: static/sass/components/header.scss [hunk 2]
```diff
@@ -63,186 +63,103 @@
...
-        content: "";
-        width: 100%;
-        height: 100%;
-        background: url('/images/hero/opensourcebox.svg') no-repeat;
-        position: absolute;
-        top: 0%;
-        right: -60%;
-        background-size: 300px;
-    }
+    background-image: linear-gradient(32deg, $themeColor 50%, 59%, $themeColorWithOpacity 50%) !important;
```
Comment (christian-bromann) [inline static/sass/components/header.scss]:
```suggestion
    background: linear-gradient(32deg, $themeColor 50%, 59%, $themeColorWithOpacity 50%) !important;
```

**Review by christian-bromann — APPROVED** [review]
LGTM

I would keep the social icons in the hero. I leave it up to you!

**Comment by diemol** [comment]
We can iterate on it and bring the icons if we feee like doing that. I think it looks cleaner now.

**Pull request merged**